tools/chapel-py/scripts/generate-pyi.py

```diff
@@ -19,237 +19,15 @@
 #
 
 import chapel.core
-from typing import List, Tuple, Optional, Union
 
 
 def _get_base_header() -> str:
     c = chapel.core.Context()
     return c._get_pyi_file()
 
 
```

Comment:
I know you are still working on this, but did you consider removing this file entirely and using the oneliner `$(CHPL_MAKE_HOME)/util/config/run-in-venv-with-python-bindings.bash python3 -c 'import chapel; print(chapel.Context()._get_pyi_file())'`. I think we have pretty much always considered this file temporary, so unless something else is going in here I think it can be removed